templates/components/slashcommand.js.sapphire

```diff
@@ -3,24 +3,41 @@
 const { Command } = require('@sapphire/framework')
 
 class UserCommand extends Command {
-    constructor(context, options) {
-        super(context, {
-            ...options,
+	/**
+	 * @param {Command.Context} context
+	 * @param {Command.Options} options
+	 */
+	constructor(context, options) {
+		super(context, {
+			...options,
 			name: 'command',
 			description: 'A basic slash command'
-        });
-    }
+		});
+	}
 
-    registerApplicationCommands(registry) {
-		registry.registerChatInputCommand({
-			name: this.name,
-			description: this.description
-		})
+	/**
+	 * @param {Command.Registry} registry
+	 */
+	registerApplicationCommands(registry) {
+		registry.registerChatInputCommand(
+			/**
+			 * @param {import('@discordjs/builders').SlashCommandBuilder} builder
+			 */
```

Comment:
This isn't needed, is it?

Reply:
Right... for some reason it didn't work before. Removed it now.